src/main/resources/db/migration/V2024_04_08_1220__CIV-11304-order_made_claimant.sql

```diff
@@ -0,0 +1,24 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.CP.OrderMade.Claimant', '{}', '{"Notice.AAA6.CP.OrderMade.Claimant" : []}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role, time_to_live)
+VALUES ('Notice.AAA6.CP.OrderMade.Claimant', 'An order has been made', 'An order has been made',
+        '<p class="govuk-body">The judge has made an order on your claim. <a href="{VIEW_FINAL_ORDER}" rel="noopener noreferrer" target="_blank" class="govuk-link">View the order</a>.</p>',
+        '<p class="govuk-body">The judge has made an order on your claim. <a href="{VIEW_FINAL_ORDER}" rel="noopener noreferrer" target="_blank" class="govuk-link">View the order</a>.</p>',
+        'CLAIMANT', 'Session');
+
+/**
+ * Add task list items
+ */
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
```

Comment:
@miguelMolina3691 Is this change needed ? If I am not wrong, in ur story, it says "View orders and notices - link is active, tag would already be Available"